package.json

```diff
@@ -3,16 +3,13 @@
   "version": "0.1.0",
   "description": "CSV grammar for tree-sitter",
   "main": "bindings/node",
-  "keywords": [
-    "parser",
-    "tree-sitter",
-    "csv"
-  ],
+  "keywords": ["parser", "tree-sitter", "csv"],
   "author": "Arnau Siches",
   "license": "MIT",
   "scripts": {
     "build": "tree-sitter generate && node-gyp build",
-    "test": "tree-sitter test"
+    "test": "tree-sitter test",
+    "dev": "npm run build && tree-sitter highlight --html examples/*.csv > target/out.html && open target/out.html"
```

Comment:
```suggestion
    "dev": "npm run build && tree-sitter highlight examples/*.csv"
```

I'd rather not have to generate html and have to manage the `target/` folder. Highlighting in the shell should suffice.

Reply:
That better to use, thanks for this tip.

<img width="1041" alt="image" src="https://github.com/arnau/tree-sitter-csv/assets/5518/c9bfae41-c61f-4803-98a7-55bb15725136">